airbyte-cdk/python/airbyte_cdk/sources/declarative/date/min_max_date.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+
+
+class MinMaxDate:
```

Comment:
I kept going back and forth on whether we should call this MinMaxDatetime to remain consistent with the slicer and for accuracy since this supports both date and time

Reply:
+1 for datetime